packages/app/src/components/choropleth/shared.d.ts

```diff
@@ -14,6 +19,11 @@ export type MetricKeys<T> = keyof Omit<
 export type MunicipalitiesMetricName = MetricKeys<Municipalities>;
 export type RegionsMetricName = MetricKeys<Regions>;
 
+export type DifferenceKeys =
```

Comment:
```suggestion
export type DifferenceKey =
```

since it refers to only one key in the end.